nearlib/test/integration.test.js

```diff
@@ -69,21 +70,54 @@ test('deploy contract and make function calls', async () => {
         "near_func_hello", // this is the function defined in hello.wasm file that we are calling
         args);
     expect(viewFunctionResult).toEqual("hello trex");
+
+    var setCallValue = await generateUniqueString("setCallPrefix");
+    const accountBeforeScheduleCall = await account.viewAccount(aliceAccountName);
+    const setArgs = {
+        "value": setCallValue
+    }
+    const scheduleResult = await nearjs.scheduleFunctionCall(
+        0,
+        aliceAccountName,
+        "test_contract",
+        "near_func_setValue", // this is the function defined in hello.wasm file that we are calling
+        setArgs);
+    waitForNonceToIncrease(accountBeforeScheduleCall);
+
+    for (let i = 0; i < TEST_MAX_RETRIES; i++) {
```

Comment:
Extract for / try / catch pattern with retries into common helper used by different tests?

Reply:
I'd like to get rid of retries eventually, but I suppose that we can have this helper for now.